src/net.rs

```diff
@@ -217,6 +238,25 @@ pub struct Response;    // TODO
 //   receiving commands, sending answers, ...)
 //
 
+#[test] 
+pub fn test_send_ok_packet() {
+    let mut vec = Vec::new();
+
+    let res = send_ok_packet(&mut vec);
+    assert_eq!(res.is_ok(), true);
+}
```

Comment:
Please add a `assert_eq!(vec, vec![Cnv::OkPkg as u8]);` to test the contents of the vector.